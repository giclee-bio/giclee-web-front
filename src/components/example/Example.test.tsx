import { render } from '@testing-library/react'

import Example from './Example'

describe('Button', () => {
  it('snapshot', async () => {
    const { asFragment } = render(<Example />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
