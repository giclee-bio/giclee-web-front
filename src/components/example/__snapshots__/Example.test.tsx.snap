// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example snapshot 1`] = `
<DocumentFragment>
  Example
</DocumentFragment>
`;
